libkbfs/md_ops_test.go

```diff
@@ -394,7 +399,7 @@ func TestMDOpsGetSuccess(t *testing.T) {
 	rmds, _ := newRMDS(t, config, false)
 
 	// Do this before setting tlfHandle to nil.
-	verifyMDForPrivate(config, rmds)
+	verifyMDForPrivateAtMostOnce(config, rmds)
```

Comment:
is there a reason you have to relax to at most once here, since it's not parallel?

Maybe have a verifyMDForPrivateHelper that takes a minTimes and a maxTimes, and have verifyMDForPrivate which calls it with 1, 1 (matching old behavior) and then you can do 0, 1 for the tests that need it.